device_local.go

```diff
@@ -43,7 +47,28 @@ func NewDevice(env BootEnvReadWriter, sc StatCommander, config deviceConfig) *de
 		active:            "",
 		inactive:          "",
 	}
+
+	if partitions.rootfsPartA == "" {
+		partitions.rootfsPartA = "mmcblk0p1"
+	}
+	if partitions.rootfsPartB == "" {
+		partitions.rootfsPartB = "mmcblk0p2"
+	}
```

Comment:
Can you elaborate? The configuration is read elsewhere and the fields with be populated with what was read at that point. This is just a fallback in case partitions were not provided in the config file.